src/components/CheckboxInput/CheckboxInput.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import { StoryObj, Meta } from '@storybook/react';
+import React from 'react';
+import { CheckboxInput } from './CheckboxInput';
+
+export default {
```

Comment:
Thanks for splitting these out! 💪 I think we can keep all the stories in the top-level file (`Checkbox.stories.tsx`) for easier discoverability. That matches their code guidelines as well